src/plan/immix/global.rs

```diff
@@ -137,13 +133,13 @@ impl<VM: VMBinding> Plan for Immix<VM> {
 
     fn prepare(&mut self, tls: VMWorkerThread) {
         self.common.prepare(tls, true);
-        self.immix_space.prepare();
+        self.immix_space.prepare(true);
     }
 
     fn release(&mut self, tls: VMWorkerThread) {
         self.common.release(tls, true);
         // release the collected region
-        self.immix_space.release();
+        self.immix_space.release(true);
```

Comment:
The return value is not used in immix. Clippy only warns about unused `Result` return value. I am not clear what your suggestion was.

Reply:
Yes, it's fine. I think maybe you can add comment to `ImmixSpace::release` to explain why this return value is added, because it diverges a bit from other space that both `prepare` and `release` generally don't have return values.